data/crac-creation/crac-creator-csa-profiles/src/test/java/com/farao_community/farao/data/crac_creation/creator/csa_profile/crac_creator/cnec/FlowCnecCreationTest.java

```diff
@@ -355,11 +359,17 @@ void testTC2FlowCnecsWithOperationalLimits() {
 
     @Test
     void testCreateCracCSATestWithRejectedFiles() {
+        ListAppender<ILoggingEvent> listAppender = getLogs(RaoBusinessWarns.class);
+        List<ILoggingEvent> logsList = listAppender.list;
+
         CsaProfileCracCreationContext cracCreationContext = getCsaCracCreationContext("/CSA_Test_With_Rejected_Files.zip");
 
+        assertEquals("[WARN] [REMOVED] The file : contexts:REE_CO.xml will be ignored. Its dates are not consistent with the current date : 2023-03-29T12:00Z", logsList.get(0).toString());
+        assertEquals("[WARN] [REMOVED] The file : contexts:ELIA_AE.xml will be ignored. Its dates are not consistent with the current date : 2023-03-29T12:00Z", logsList.get(1).toString());
+
```

Comment:
Not sure this test is reproducible :thinking: could you sort the logList first please?